subprojects/logging/src/main/java/org/gradle/internal/logging/events/RenderableOutputEvent.java

```diff
@@ -16,13 +16,16 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 import org.gradle.internal.logging.text.StyledTextOutput;
 
 public abstract class RenderableOutputEvent extends CategorisedOutputEvent {
+    private final OperationIdentifier operationId;
```

Comment:
Should `OperationIdentifier` become something more general that in the long run could resemble attributes of a `BuildOperation`? I can think of at least one more attribute that we'll need to add soon: the category to indicate that it came from a task or something else.